### PR TITLE
chore: upgrade blackhole dependency

### PR DESCRIPTION
**Pull request by 3Hren**

### CODE REVIEW

**Comment by 3Hren** [comment]
Also testing nazi-PR-guard to prevent unreviewed merges.

**Comment by 3Hren** [comment]
@antmat say LGТM please.

**Comment by antmat** [comment]
НБМХ

**Comment by antmat** [comment]
НБМХ

**Comment by antmat** [comment]
Looks good to me

**Comment by antmat** [comment]
E-lle-ven!

**Comment by antmat** [comment]
A-lleven!

**Comment by 3Hren** [comment]
LGTM!

**Comment by 3Hren** [comment]
LGTM

**Comment by 3Hren** [comment]
Пиздец неработающее говно.

**Comment by antmat** [comment]
LGTM?

**Comment by 3Hren** [comment]
Я про сервис

File: debian/control [hunk 1]
```diff
@@ -9,7 +9,7 @@ Build-Depends: cmake, cdbs, debhelper (>= 7.0.13), libltdl-dev, libmsgpack-dev,
  libboost-thread-dev,
  libboost-program-options-dev,
  metrics-dev,
- blackhole-dev (>= 1.0.0-0rc1)
+ blackhole-dev (>= 1.0.0)
```
Comment (shaitan) [inline debian/control]:
```bash
$ dpkg --compare-versions 1.0.0-0rc1 gt 1.0.0 && echo true
true
```

Upgrade it to `>= 1.0.0-1`.

**Pull request closed without merging**